src/imports.jl

```diff
@@ -190,7 +191,7 @@ import ITensors.NDTensors:
   addblock!,
   store
 
-using NDTensors.Unwrap: unwrap_type
+using NDTensors.Unwrap: unwrap_array_type
```

Comment:
I think it makes sense to move `unwrap_array_type` to `TypeParameterAccessors`, since that is where `is_wrapped_array` is defined now.

Reply:
Hmm, then should we rename `Unwrap` to `Expose`?

Reply:
Good question, it probably makes sense to change the name of `Unwrap` if we move `unwrap_array_type` to `TypeParameterAccessors`. `Expose` seems like too general of a name, maybe `ExposeArrayWrappers`?